test_trained_model.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
```

Comment:
I don't think we need the header (mostly this would be the only file that has it)